pkg/engine/executor/parse_logfmt.go

```diff
@@ -34,46 +34,13 @@ func BuildLogfmtColumns(input *array.String, requestedKeys []string, allocator m
 // parseKeys discovers columns dynamically as lines are parsed
 func parseKeys(input *array.String, requestedKeys []string, columnBuilders map[string]*array.StringBuilder, allocator memory.Allocator) []string {
 	columnOrder := []string{}
-	var errorBuilder, errorDetailsBuilder *array.StringBuilder
-	hasErrorColumns := false
 
 	for i := 0; i < input.Len(); i++ {
 		line := input.Value(i)
-		parsed, err := tokenizeLogfmt(line, requestedKeys)
-
-		// Handle error columns
```

Comment:
I don't think we should remove this logic. We can handle skipping `__error__` and `__error_details__` in the compat layer